mmdet3d/models/model_utils/img_feat_mlp_module.py

```diff
@@ -0,0 +1,48 @@
+from mmcv.cnn import ConvModule
+from torch import nn as nn
+
+
+class ImageMLPModule(nn.Module):
+    """Image MLP module.
+
+    Pass the image vote features through an MLP.
+
+    Args:
+        in_channels (int): Number of channels of image vote features.
+            Default: 18.
+        conv_channels (tuple[int]): Out channels of the convolution.
+            Default: (256).
+        conv_cfg (dict): Config of convolution.
+            Default: dict(type='Conv1d').
+        norm_cfg (dict): Config of normalization.
+            Default: dict(type='BN1d').
+        act_cfg (dict): Config of activation.
+            Default: dict(type='ReLU').
+    """
+
+    def __init__(self,
+                 in_channel=18,
+                 conv_channels=(256),
```

Comment:
(256, ) for tuple? Otherwise, the default input may trigger error (it will be regarded as int?).